actions.go

```diff
@@ -87,12 +87,12 @@ type CustomActionsTriggerDefinitionCreateInput struct {
 }
 
 type CustomActionsTriggerDefinitionUpdateInput struct {
-	Id                     graphql.ID                                      `json:"id"`
+	Id                     ID                                              `json:"id"`
 	Name                   *graphql.String                                 `json:"name,omitempty"`
 	Description            *graphql.String                                 `json:"description,omitempty"`
-	Owner                  interface{}                                     `json:"ownerId,omitempty"`
-	Action                 interface{}                                     `json:"actionId,omitempty"`
-	Filter                 interface{}                                     `json:"filterId,omitempty"`
+	Owner                  *ID                                             `json:"ownerId"`
```

Comment:
I think we want `omitempty` still for these because they are optional

Reply:
Yeah, I figured it would be ok if we passed null for these so I didn't know if we wanted it to look more like the update struct for consistency or if we wanted to just omit the fields.

Reply:
Oh sorry, this is on the update struct... I thought we decided we wanted to go with option 3 from the scalar tests you wrote and work out the logic on the terraform side?

Reply:
option 3 should be the least likely one - for the most part for referencing IDs of other resources i think `Create` will use number 2 and `Update` will use number 4.

1 is likely needed for ID's that are the resource identifier itself.  I cannot think of an usecase where number 3 is actually needed.

Reply:
Ah gotcha, I must have mixed them up.  I'll make those changes now.